Documentation/cross-platform-performance-and-eventing.md

```diff
@@ -0,0 +1,287 @@
+# .NET Cross-Plat Performance and Eventing Design
+
+##Introduction
+
+As we bring up CoreCLR on the Linux and Mac platforms, it’s important that we determine how we’ll measure and analyze performance on these platforms.  On Windows we use an event based model that depends on ETW, and we have a good amount of tooling that builds on this approach.  Ideally, we can extend this model to Linux and Mac and re-use much of the Windows tooling.
+
```

Comment:
I've definitely been thinking about this some.  Unfortunately in many cases, technology choices for operating systems are mutually exclusive.  The good news is that for most performance data, there is no work to do in the runtime.  Most of the runtime work is specific to tracing, and the tracing design will be pluggable for this very reason.